jscode.js

```diff
@@ -0,0 +1,22 @@
+var words = ['It', 'is', 'awfully', 'hard'];
+var times = [1000, 2000, 4000, 6000];
+
+function showAyah(words,times){
+
+	var htmlElement = document.querySelector(".ayahWords");
+	let wordCount = words.length;

```

Comment:
Suppose, you made an error somewhere and passed incorrect `words` and `times` to the function. You want to catch this error as soon as possible. Add a check, that if the lengths of these two arrays different, then throw an exception. You can do it as follows:
```js
if (words.length !== times.length) {
    throw "Words and times lengths must be equal!";
}
```